providers/default.rb

```diff
@@ -107,25 +107,29 @@ def upgradeable?(name)
   end
 
   Chef::Log.debug("Checking to see if this chocolatey package is installed/upgradable: '#{name}'")
-  cmd = Mixlib::ShellOut.new("#{::ChocolateyHelpers.chocolatey_executable} version #{name} #{cmd_args}")
+  cmd = Mixlib::ShellOut.new("#{::ChocolateyHelpers.chocolatey_executable} upgrade -r --noop #{cmd_args} #{name}")
   cmd.run_command
-  !cmd.stdout.include?('Latest version installed')
+  result = cmd.stdout.chomp
+  return false if result.empty? # for chocolatey 0.9.9 + 0.9.9.1
```

Comment:
I don't have any big problem about being compatible only with a newer version of chocolatey.

Any user who want the "old" behaviour can pin his cookbook (And he really should use pessimistic versionning!)

We only have to push a new cookbook with minor updated